Lib/test/test_asyncio/test_timeouts.py

```diff
@@ -116,15 +119,57 @@ async def test_foreign_exception_passed(self):
                 raise KeyError
         self.assertFalse(cm.expired())
 
+    async def test_timeout_exception_context(self):
+        with self.assertRaises(TimeoutError) as cm:
+            async with asyncio.timeout(0.01):
+                try:
+                    1/0
+                finally:
+                    await asyncio.sleep(1)
+        e = cm.exception
```

Comment:
Maybe add a comment? The structure we expect is `TimeoutError` caused by `CancelledError` caused by `ZeroDivisionError` caused by `None`.